plugin-server/src/main/utils.ts

```diff
@@ -57,19 +58,22 @@ export async function kafkaHealthcheck(
             ],
         })
 
-        consumer.resume([{ topic: KAFKA_HEALTHCHECK }])
```

Comment:
Note that this was also racy before - we might call `FETCH_START` too fast.

Reply:
fair point!